03-Custom-Actions-In-Python/06-Improving-Performance.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>You'll need to explicitly seed the numpy random number generator so that all ranks choose the same particle tag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good catch.</p>